sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/options/BlobInputStreamOptions.java

```diff
@@ -86,4 +89,25 @@ public BlobInputStreamOptions setConsistentReadControl(ConsistentReadControl con
         this.consistentReadControl = consistentReadControl;
         return this;
     }
+
+    /**
+     * Get the timeout value for how long the client should wait to get the headers on each download request issued.
+     * Does not include the time needed to read the body.
+     * @return The timeout value.
```

Comment:
do we have an ask for this functionality ? It'd be better to point users towards `HttpClientOptions` or `HttpClient` builders to set global timeouts unless there's good reason to add this.